marketplace_builder/views/pages/tutorials/search/filtering-sorting.liquid

```diff
@@ -0,0 +1,85 @@
+---
+converter: markdown
+metadata:
+  title: Filtering and Sorting
+  description: Filtering and sorting using GraphQL queries.
+slug: tutorials/search/filtering-sorting
+searchable: true
+---
+
+Filtering using Elasticsearch is a slightly different operation from [Search](/tutorials/search/search), but they are easy to confuse. This topic explains filtering and sorting using GraphQL queries. 
+
+## Filtering
+
+To filter items by exact value use
+
+* `user:` for `people`
+
+```graphql
+people(user: {first_name: "darek"}) {
+  results {
+    email
+  }
+}
+```
+
+
+* `listing` for `listings`
+
+```graphql
+listings(listing: {slug: "transactable-slug"}){
+  results {
+    slug
+  }
+}
+``` 
+
+Please note, that there is no filter argument in the customizations entry point. You specify filtering operations directly. 
+
+```graphql
+{
+  customizations(
+    page: 1 per_page: 210
+    properties: [
+      { name: "name", value: "alpha|beta" }
+    ]
+    name: "company"
+  ) {
+    total_entries
+    results {
+      id
+      name
+      properties
+    }
+  }
+}
+```
+
+## Sorting
+
+Use `sort` to change your the ordering of your collection. 
+When sorting automatically, the `raw` field is used. 
+
+```graphql
+{
+  customizations(
+    page: 1 per_page: 210
```

Comment:
```suggestion
    page: 1
    per_page: 210
```

Reply:
@pavelloz no commas are correct and it's correct graphql syntax but i agree for better readability it's OK to change it

Reply:
im ok with both, but everywhere the same